Resources/Private/Fusion/Integration/Footer.fusion

```diff
@@ -13,6 +13,7 @@ prototype(Neos.Demo:Integration.Footer.Renderer) < prototype(Neos.Demo:Presentat
     menuItems = Neos.Fusion:Map {
         items = ${q(site).property('metaNavigationItems')}
         itemRenderer = Neos.Fusion:DataStructure {
+            depth = ${node.property('_depth')}
```

Comment:
`_depth` is also deprecated and should better not be used in the field.
Still deprecated but better is `\Neos\Neos\Fusion\Helper\NodeHelper::depth`

Reply:
why is the depth needed here at all?

Reply:
It isn't, I think it was in the past? IDK, removing